sdk/batch/azure-batch/azure/batch/models/_models.py

```diff
@@ -539,7 +539,7 @@ def __init__(self, **kwargs):
 
 
 class BatchErrorException(HttpOperationError):
-    """Server responsed with exception of type: 'BatchError'.
+    """Server responded with exception of type: 'BatchError'.
```

Comment:
This file is generated via a tool called autorest. We don't edit generated code files since they will just be overwritten during the next generation. You can tell if a file is generated by looking at the header:

https://github.com/Azure/azure-sdk-for-python/blob/e0523d0778aa8afd7863faaecdf73fdce2195720/sdk/batch/azure-batch/azure/batch/models/_models.py#L7-L9

That being said, can we revert any changes to generated files in this PR? You can add the misspelled words to the cspell.json for now.

Reply:
I'm sorry. I was just thinking of fixing the typo. I revert the _models.py, _models_py3.py.